### PR TITLE
Remove employment status 

### PR DESCRIPTION
**Pull request by hiboabd**
## Description of change
Agreed to remove as no longer required

### CODE REVIEW

**Review by timpeat — APPROVED** [review]
🧹

File: spec/fixtures/application/1.0/application_returned.json [hunk 1]
```diff
@@ -29,11 +29,7 @@
       "telephone_number": "",
       "correspondence_address_type": "providers_office_address",
       "home_address": null,
-      "correspondence_address": null,
-      "income_details": {
-        "employment_type": ["not_working"],
```
Comment (arthurashman) [inline spec/fixtures/application/1.0/application_returned.json:34]: This one we want to keep I think!

File: spec/fixtures/application/1.0/application_returned.json [hunk 2]
```diff
@@ -29,11 +29,7 @@
...
-        "ended_employment_within_three_months": "no"
```
Comment (arthurashman) [inline spec/fixtures/application/1.0/application_returned.json:35]: And this!

Comment (arthurashman) [inline spec/fixtures/application/1.0/application_returned.json:35]: oops, thank you! 🧹

**Pull request merged**